### ISSUE TITLE
LUA Error: attempt to index field 'specSpells' (a nil value)

### ISSUE THREAD

**Issue by MarcelMertens** [corporategoth/rotationmaster#7]
Hi,

when adding a spell to my rotation (Enter "Seal of Rig.." -> Select the Spell from the Dropdown) i'm getting this error:
![image](https://user-images.githubusercontent.com/20799825/181683079-f9b1becb-b79a-4de1-8e56-493dafb29bb9.png)

```
Date: 2022-07-29 06:28:58
ID: 1
Error occured in: Global
Count: 1
Message: ..\AddOns\RotationMaster\utils.lua line 289:
   attempt to index field 'specSpells' (a nil value)
Debug:
   [string "=(tail call)"]: ?
   [string "@RotationMaster\utils.lua"]:289:
      RotationMaster\utils.lua:283
   [string "@RotationMaster\Options/rotations.lua"]:477:
      ...nterface\AddOns\RotationMaster\Options/rotations.lua:474
   [string "=[C]"]: ?
   [string "@Masque\Libs\AceGUI-3.0\AceGUI-3.0.lua"]:72:
      ...terface\AddOns\Masque\Libs\AceGUI-3.0\AceGUI-3.0.lua:70
   [string "@Masque\Libs\AceGUI-3.0\AceGUI-3.0.lua"]:306: Fire()
   [string "@RotationMaster\Libs\AceGUI-3.0-Spell-EditBox\EditBox-Core.lua"]:421:
      ...aster\Libs\AceGUI-3.0-Spell-EditBox\EditBox-Core.lua:410
Locals:
None
AddOns:
  Swatter, v2.5.6773 (SwimmingSeadragon)
  AdiBags, vv1.9.26-bcc
  AdiBagsBetterCategories, v0.3.1-tbc
  AdiBagsItemRackSets, v@project-version@
  AdiBagsPT3Filter, vv1.4.10-bcc
  Altoholic, v2.5.008
  Atlas, vv1.51.00
  AtlasBurningCrusade, vv1.51.00
  AtlasClassicWoW, vv1.51.00
  AtlasDungeonLocs, vv1.51.00
  AtlasTransportation, vv1.51.02
  AtlasLootClassic, vv2.4.4-bcc
  AtlasLootClassicData, vv2.4.4-bcc
  AtlasLootClassicDungeonsAndRaids, vv2.4.4-bcc
  AucAdvanced, v2.5.6733 (SwimmingSeadragon)
  AucFilterBasic, v2.5.6759 (SwimmingSeadragon)
  AucStatHistogram, v2.5.6761 (SwimmingSeadragon)
  AucStatiLevel, v2.5.6765 (SwimmingSeadragon)
  AucStatPurchased, v2.5.6762 (SwimmingSeadragon)
  AucStatSimple, v2.5.6763 (SwimmingSeadragon)
  AucStatStdDev, v2.5.6764 (SwimmingSeadragon)
  BagSync, v16.7
  Bartender4, v4.11.2
  BeanCounter, v2.5.6767 (SwimmingSeadragon)
  DataStore, v2022.06.05
  DataStoreAgenda, v2.5.008
  DataStoreAuctions, v2.5.008
  DataStoreCharacters, v2022.06.05
  DataStoreContainers, v2.5.008
  DataStoreCrafts, v2.5.008
  DataStoreInventory, v2.5.008
  DataStoreMails, v2022.06.05
  DataStoreQuests, v2.5.008
  DataStoreReputations, v2.5.008
  DataStoreSpells, v2022.06.05
  DataStoreTalents, v2.5.008
  DBMCore, vee0b66f
  DBMStatusBarTimers, vee0b66f
  DBMVPVEM, vbffd391
  Decursive, v2.7.8.6
  Details, v
  DetailsStreamer, v
  DetailsTinyThreat, v
  Enchantrix, v2.5.6768 (SwimmingSeadragon)
  EnchantrixBarker, v2.5.6769 (SwimmingSeadragon)
  ExtraTooltip, v1.16
  GatherMate2, v1.46.1-classic
  Guidelime, v2.029
  GuidelimeFlymolosOutlandLevelingGuide, v2.0.2-Release
  HidingBar, vv2.5.33
  Informant, v2.5.6770 (SwimmingSeadragon)
  ItemRack, v
  LeatrixMaps, v2.5.116
  LeatrixPlus, v2.5.116
  LFGBulletinBoard, v2.61
  Mapster, v1.9.7
  Masque, v9.2.6
  MikScrollingBattleText, v5.7.153
  NovaInstanceTracker, v1.29
  NovaRaidCompanion, v1.17
  NovaWorldBuffs, v2.28
  PallyPower, vv1.4.5-classic
  Postal, vv3.8.1
  Prat30, v3.9.1
  Prat30Libraries, v
  ProfessionAssistance, v4.0.3
  Questie, v6.13.1
  RotationMaster, v1.3.9
  Scrap, v9.1.2
  ScrapConfig, v
  SexyMap, vv9.2.6
  ShadowedUnitFrames, vv4.3.2-classic
  SlideBar, v2.5.6771 (SwimmingSeadragon)
  Stubby, v2.5.6772 (SwimmingSeadragon)
  Talented, vv211202
  TalentedData, v
  ThreatClassic2, v2.44
  TidyPlatesThreatPlates, v10.4.1
  TomTom, vv3.0.7
  TrinketMenu, v9.0.7
  WeakAuras, v4.0.2
  WideQuestLog, v
  WideQuestLogLevels, v
  BlizRuntimeLib_enUS v2.5.4.20504 <none>
  (ck=9b9)

```

Any Idea?

**Comment by corporategoth** [comment]
Confirm if this is fixed in 1.4.0-beta2 please?

**Comment by corporategoth** [comment]
Should definitely be fixed in 1.4.0-beta3
